index.html

```diff
@@ -259,6 +259,15 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
       </section>
     </section>
   </div>
-       </section>
+</section>
+<footer class="form-cont">
+  <form action="https://formspree.io/f/mrgdkyla" method="post">
```

Comment:
- [ ] I get the error in the image below when I submit a form. I kindly suggest that you confirm that the form is submitted successfully with valid input values.


<img width="302" alt="image" src="https://user-images.githubusercontent.com/52934128/193897646-3b3176da-e451-4b2a-9403-d01bf15bc913.png">